src/test/java/com/rackspacecloud/metrics/ingestionservice/IngestionServiceApplicationTests.java

```diff
@@ -32,65 +33,59 @@
+  private Sender sender;
 
 	@MockBean
-    private InfluxDBHelper influxDBHelperMock;
-
-	@Before
-    public void setup(){
-	    // Nothing to setup at this time.
+  private InfluxDBHelper influxDBHelperMock;
+
+  @Test
+  @Ignore
+  public void testSuccessfulRawDataConsumption() throws IOException, InterruptedException {
+      // Mock influxDB ingestion call
+    doNothing().when(this.influxDBHelperMock).ingestToInfluxDb(anyString(), anyString(), anyString(), anyString());
+
+    for(int i = 0; i < 1; i++) {
+      sender.send(MockMetricHelper.getValidMetric(
+            i,
+            "CORE",
+            "hybrid:1667601",
+            13,
+            true),
+          UNIFIED_METRICS_TOPIC);
     }
 
-	@Test
-    @Ignore
-	public void contextLoads() {
-	}
-
-	@Test
-    @Ignore
-    public void testSuccessfulRawDataConsumption() throws Exception {
-        // Mock influxDB ingestion call
-		doNothing().when(this.influxDBHelperMock).ingestToInfluxDb(anyString(), anyString(), anyString(), anyString());
-
-	    for(int i = 0; i < 1; i++) {
-            sender.send(
-                    MockMetricHelper.getValidMetric(i, "CORE", "hybrid:1667601",
-                            13, true),
-                    UNIFIED_METRICS_TOPIC);
-        }
-
-        Thread.sleep(10*1000L); // wait for a few sec for consumer to process some records
-
-        long batchProcessed = rawListener.getBatchProcessedCount();
-        Assert.assertTrue(batchProcessed > 0);
-    }
-
-    @Test
-    @Ignore
-    public void test_whenIngestToInfluxDBThrowsException_globalExceptionHandlerCatches() throws Exception {
-        // Mock influxDB ingestion call
-        doThrow(new Exception("test_whenIngestToInfluxDBThrowsException_globalExceptionHandlerCatches"))
-				.when(this.influxDBHelperMock).ingestToInfluxDb(anyString(), anyString(), anyString(), anyString());
-
-        for(int i = 0; i < 1; i++) {
-            sender.send(
-                    MockMetricHelper.getValidMetric(i, "CORE", "hybrid:1667601",
-                            11, true),
-                    UNIFIED_METRICS_TOPIC);
-        }
-
-        Thread.sleep(10*1000L); // wait for a few sec for consumer to process some records
-
-        // Batch processed count will still be more than 0 because exception thrown doesn't
-        // mean that batch is not processed
-        long batchProcessed = rawListener.getBatchProcessedCount();
-        Assert.assertTrue(batchProcessed > 0);
-    }
+      Thread.sleep(10*1000L); // wait for a few sec for consumer to process some records
+
+      long batchProcessed = rawListener.getBatchProcessedCount();
+      Assert.assertTrue(batchProcessed > 0);
+  }
+
+  @Test
+  @Ignore
+  public void test_whenIngestToInfluxDBThrowsException_globalExceptionHandlerCatches()
+      throws IOException, InterruptedException {
+      // Mock influxDB ingestion call
+      doThrow(new IOException("test_whenIngestToInfluxDBThrowsException_globalExceptionHandlerCatches"))
```

Comment:
This thrown exception doesn't seem to get aserted; however, it'll be hard for you to tell since the unit test is `@Ignore`d